### PR TITLE
DBI-519 : Suppressing unnecessary errors/warning in RDF parsing

### PR DESCRIPTION
**Pull request by abika5**

### CODE REVIEW

File: src/main/java/com/marklogic/contentpump/RDFReader.java [hunk 1]
```diff
@@ -1044,8 +1044,12 @@ private String formatMessage(String message, long line, long col) {
 
         @Override
         public void warning(String message, long line, long col) {
-            //For Bug 24519, to improve readability of logs, I have chose to log the warnings for IRI only in debug mode. 
-            if (message.contains("Bad IRI:") || message.contains("Illegal character in IRI")) {
+            //For Bug 24519, to improve readability of logs, I have chose to log the warnings for IRI only in debug mode.
+            List<String> debugwarninglist = new ArrayList<>();
```
Comment (yunzvanessa) [inline src/main/java/com/marklogic/contentpump/RDFReader.java:1048]: Please note the Java naming convention here. debugwarninglist --> debugWarningList, debugerrorlist --> debugErrorList

Comment (yunzvanessa) [inline src/main/java/com/marklogic/contentpump/RDFReader.java:1048]: One suggestion is to use HashSet other than List, due to the search performance.

**Review by yunzvanessa — CHANGES_REQUESTED** [review]

File: src/main/java/com/marklogic/contentpump/RDFReader.java (hunk 1 above)
Comment (abika5) [reply src/main/java/com/marklogic/contentpump/RDFReader.java:1048]: Excellent point, will do it.

Comment (abika5) [reply src/main/java/com/marklogic/contentpump/RDFReader.java:1048]: Got it, will do it.

**Pull request closed without merging**